mobilizon_reshare/main/pull.py

```diff
@@ -0,0 +1,25 @@
+import logging.config
+
+from mobilizon_reshare.event.event import MobilizonEvent
+from mobilizon_reshare.mobilizon.events import get_mobilizon_future_events
+
+from mobilizon_reshare.storage.query.write import (
+    create_unpublished_events,
+)
+
+logger = logging.getLogger(__name__)
+
+
+async def pull() -> list[MobilizonEvent]:
```

Comment:
You should add dedicated tests for the command and how it interacts with start. You should at least test:

* Pull with no events anywhere
* Pull with one event on mobilizon and none on the db
* Pull with one event on mobilizon and another on the db
* Pull with one event on mobilizon and the same event on the db
* sequence: pull->start
* Sequence: start->pull->
* Sequence: pull->new event on mobilizon -> pull

Reply:
fixed, thanks